src/test/java/org/jpeek/metrics/MetricBase.java

```diff
@@ -0,0 +1,150 @@
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.jpeek.metrics;
+
+import com.jcabi.xml.XML;
+import com.jcabi.xml.XSLDocument;
+import org.cactoos.io.ResourceOf;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.hamcrest.number.IsCloseTo;
+import org.jpeek.FakeBase;
+import org.jpeek.skeleton.Skeleton;
+
+/**
+ * Metric test helper.
+ * @author Ilya Kharlamov (ilya.kharlamov@gmail.com)
+ * @version $Id$
+ * @since 0.28
+ */
+public final class MetricBase {
+    /**
+     * XSL document.
+     */
+    private final XSLDocument xsl;
+
+    /**
+     * Ctor.
+     * @param path Path to the xsl
+     * @throws Exception If file not found.
+     */
+    public MetricBase(final String path) throws Exception {
+        this.xsl = new XSLDocument(
+            new ResourceOf(
+                path
+            ).stream()
+        );
+    }
+
+    /**
+     * Transform a class to assertable xml.
+     * @param name File name (without an extension) of a class to transform
+     * @return Xml result of the transformation
+     */
+    public Report transform(final String name) {
+        return new Report(
+            name,
+            this.xsl.transform(
+                new Skeleton(
+                    new FakeBase(name)
+                ).xml()
+            )
+        );
+    }
+
+    /**
+     * Assertion helper for xml.
+     */
+    public static final class Report {
+        /**
+         * Copy of the transformation xml.
+         */
+        private final XML xml;
+
+        /**
+         * Class name.
+         */
+        private final String name;
+
+        /**
+         * Ctor.
+         * @param name Class name.
+         * @param xml Resulting xml.
+         */
+        public Report(final String name, final XML xml) {
+            this.name = name;
+            this.xml = xml;
+        }
+
+        /**
+         * Asserts the variable produced.
+         * @param variable Variable name
+         * @param expected Expected value
+         */
+        public void assertVariable(final String variable, final int expected) {
+            MatcherAssert.assertThat(
+                String.format(
+                    "Variable '%s' is not calculated correctly for class '%s'",
+                    variable,
+                    this.name
+                ),
+                this.xml.xpath(
+                    String.format(
+                        "//class[@id='%s']/vars/var[@id='%s']/text()",
+                        this.name,
+                        variable
+                    )
+                ).get(0),
+                new IsEqual<>(
```

Comment:
@ilyakharlamov Use `TextIs` instead.

Reply:
@fabriciofx `TextIs` is cactoos-matchers, we don't have this dependency in jpeek project. Shall I add it?

Reply:
@paulodamaso done in 63e7dc8